src/tools/singlejar/diag.h

```diff
@@ -19,7 +19,7 @@
  * Various useful diagnostics functions from Linux err.h file, wrapped
  * for portability.
  */
-#if defined(__APPLE__) || defined(__linux)
+#if defined(__APPLE__) || defined(__linux__)
```

Comment:
How is this change related to supporting power? Also, are we sure the case `defined(__linux)` was wrong in the first place and is not needed any more?